firmware/thruna/DIM/src/cubemx/Src/main.c

```diff
@@ -288,6 +288,37 @@ static const GlobalsConfig globals_config = {
     .dim_status_led = &dim_status_led,
 };
 
+// config to forward can functions to shared heartbeat
+// DIM rellies on BMS
+bool heartbeatMonitorChecklist[HEARTBEAT_BOARD_COUNT] = { [BMS_HEARTBEAT_BOARD] = true,
+                                                          [DCM_HEARTBEAT_BOARD] = false,
+                                                          [PDM_HEARTBEAT_BOARD] = false,
+                                                          [FSM_HEARTBEAT_BOARD] = false,
+                                                          [DIM_HEARTBEAT_BOARD] = false };
```

Comment:
DIM should rely on every board, since it displays the faults/warning of every board